srcs/layouts/latn_azerty_be.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Adapted from Belgian AZERTY -->
+<keyboard name="AZERTY (Belgian)" script="latin">
+  <row>
+    <key key0="a" key2="1" key4="loc esc"/>
+    <key key0="z" key1="|" key2="2" key3="&amp;" key4="\@"/>
+    <key key0="e" key2="3" key3="&quot;" key4="\#"/>
+    <key key0="r" key2="4" key3="'"/>
+    <key key0="t" key2="5" key3="(" key4=")"/>
+    <key key0="y" key2="6" key3="§" key4="^"/>
+    <key key0="u" key2="7" key3="{" key4="}"/>
+    <key key0="i" key2="8" key3="!" key4="ç"/>
+    <key key0="o" key1="loc accent_ogonek" key2="9" key3="à" key4="loc accent_cedille"/>
+    <key key0="p" key2="0" key3="-" key1="_"/>
+  </row>
+  <row>
+    <key key0="q" key2="loc tab"/>
+    <key key0="s" key2="ß"/>
+    <key key0="d" key2="é"/>
+    <key key0="f"/>
+    <key key0="g"/>
+    <key key0="h" key1="[" key2="]"/>
+    <key key0="j" key1="è" key2="accent_trema" key3="accent_circonflexe"/>
+    <key key0="k" key1="€" key2="*" key3="$"/>
+    <key key0="l" key1="loc accent_caron" key2="%" key3="ù" key4="accent_aigu"/>
+    <key key0="m" key1="loc accent_breve" key2="£" key3="µ" key4="accent_grave"/>
```

Comment:
The keyboard contains àèù, is `accent_grave` useful ? For occasionally writing other accented letters, isn't the compose key enough ?
It's also possible to make this key appear only if the user has a language that needs it installed on their device.

Same for `accent_aigu` and `accent_cedille`, éç are already on the keyboard.